### DISCUSSION TITLE
Can't open my codespace

### DISCUSSION THREAD

**Discussion by Kevinklein727** [community/community#62627] [Codespaces]
### Select Topic Area

Bug

### Body

I can't open my CS50 workspace. I'm getting this error:

The workbench failed to connect to the server (Error: connection record disposed by ConnectionHub: {
"errorMessage": "Failed to start VS Code Remote server.",
"originalErrorType": "<no original type>",
"errorCode": -1,
"errorType": "GenericError",
"stack": "Error: Failed to start VS Code Remote server.\n at [CODESPACE_URL]\n at [CODESPACE_URL]\n at [CODESPACE_URL]\n at [CODESPACE_URL]\n at d ([CODESPACE_URL]\n at a ([CODESPACE_URL]"
} $1 $1)

Any help would be appreciated.

**Comment by hayataarrass** [comment]
same for me, did you figure it out?

**Comment by hayataarrass** [comment]
figured it out! go to https://cs50.dev/restart to restart the codespace and it worked for me

**Comment by Kevinklein727** [comment]
yes I got it too that way!
Thank you :)
